src/app/space/[did]/root/[cid]/page.tsx

```diff
@@ -39,9 +40,23 @@ export default function ItemPage ({ params }: PageProps): JSX.Element {
     onError: err => console.error(err.message, err.cause)
   })
 
+  const [isRemoveConfirmModalOpen, setRemoveConfirmModalOpen] = useState(false)
+  const router = useRouter()
+  const { mutate } = useSWRConfig()
+
   if (!space) {
     return <h1>Space not found</h1>
   }
+
+  const handleRemove = async () => {
+    await client?.remove(root, { shards: true })
+    setRemoveConfirmModalOpen(false)
+    // ensure list data is fresh
+    mutate(`/space/${spaceDID}/uploads?cursor=&pre=`)
```

Comment:
would be nice to pull this key string generation out to a function that can also be used here: https://github.com/web3-storage/console/blob/main/src/app/space/%5Bdid%5D/page.tsx#L25